.github/stale.yml

```diff
@@ -0,0 +1,38 @@
+# Number of days of inactivity before an Issue becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale. Set to `[]` to disable
+# We don't close any issue that is an enhancement or confirmed bug, but issues
+# waiting for reproduction cases and questions tend to get outdated.
+exemptLabels:
+  - "enhancement"
+  - "bug"
+  - "thinking"
+  - "docs"
+
+# Label to use when marking as stale
+staleLabel: "waiting-reply"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: |
+  Hey there,
+  We wanted to check in on this request since it has been inactive for 90 days. 
+  Are you having the same issue with the latest version of Raft? Have you reviewed the latest [godocs](https://godoc.org/github.com/hashicorp/raft)? 
```

Comment:
This could sound to some a little too forward like we are expecting users to have been keeping up to date.

Perhaps we could word it more like:

> If you think this is still an important issue in the latest version of [the raft library](...) or [its documentation](...) please feel let us know what you are seeing now and we'll keep it open for investigation.